package.json

```diff
@@ -30,7 +31,9 @@
   "devDependencies": {
     "audio-generator": "^1.0.5",
     "audio-speaker": "^1.1.2",
-    "watchify": "^3.6.0"
+    "mic": "^2.1.1",
+    "watchify": "^3.6.0",
```

Comment:
> there is CORS-limitations on running files/mic from locally opened file

in Chrome at least, I'm able to click the `examples/microphone-browser/index.html` file, click "allow" from the "use microphone" popup dialog, and see the waveform render.